### PR TITLE
Added a note about log file sizes and the use of logrotate

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.3+ |
| Fixed tickets | #1161 |

### CODE REVIEW

**Comment by cordoval** [comment]
is it customary to link ```logrotate``` to some linux official link?

File: cookbook/logging/monolog.rst [hunk 1]
```diff
@@ -229,8 +229,14 @@ Monolog allows you to process the record before logging it to add some
 extra data. A processor can be applied for the whole handler stack or
 only for a specific handler.
 
-A processor is simply a callable receiving the record as its first argument.
+.. tip::
 
+    Beware that log file sizes can grow very rapidly, leading to disk space exhaustion.
+    This is specially true in the ``dev`` environment, where a simple request can
+    generate hundreds of log lines. Consider using tools like the ``logrotate``
+    Linux command to contain log files before they become a problem.
```
Comment (xabbuh) [inline cookbook/logging/monolog.rst]: to rotate log files

**Comment by timglabisch** [comment]
i think this topic is very important. shouldn't we provide an example logrotate configuration?

**Comment by stof** [comment]
@timglabisch I don't think documenting logrotate is in the scope of the symfony doc. Linking to the logrotate doc seems a better idea.

**Comment by javiereguiluz** [comment]
Anybody knows which is the official web site of `logrotate`?

**Comment by xabbuh** [comment]
Looking at what is provided in the `logrotate` manpage, there's seems to be nothing else than https://fedorahosted.org/logrotate/.

**Comment by javiereguiluz** [comment]
@xabbuh thanks for the reference. I've just linked the `logrotate` command to that URL.

**Comment by xabbuh** [comment]
:+1: Looks good to me. By the way, how is this handled on Windows?

**Comment by weaverryan** [comment]
Thanks Javier! I still think we should mention "rotating_file" as a backup - it's not as robust, but is much easier to get setup for the majority of people. I'd welcome a PR mentioning it (I didn't personally know it existed for years). Cheers!

**Pull request merged**